src/oc/email/mailer.clj

```diff
@@ -44,16 +47,42 @@
                     {:html body})
               to))))
 
+(defn- inline-css [html-file inline-file]
+  (shell/sh "juice" 
+            "--web-resources-images" "false"
+            "--remove-style-tags" "true"
+            "--preserve-media-queries" "false"
+            "--preserve-font-faces" "false"
+            html-file inline-file))
+
 (defn send-snapshot
   "Create an HTML snapshot and email it to the specified recipients."
-  [{note :note snapshot :snapshot :as msg}]
-  (let [uuid-fragment (subs (str (java.util.UUID/randomUUID)) 0 4)
+  [{note :note snapshot :snapshot origin-url :origin-url :as msg}]
+  (let [full-snapshot (-> snapshot 
+                        (assoc :note note)
+                        (assoc :origin-url origin-url))
+        uuid-fragment (subs (str (java.util.UUID/randomUUID)) 0 4)
         html-file (str uuid-fragment ".html")
         inline-file (str uuid-fragment ".inline.html")]
     (try
-      (spit html-file (content/snapshot-html (assoc snapshot :note note))) ; create the email in a tmp file
-      (shell/sh "juice" "--web-resources-images" "false" html-file inline-file) ; inline the CSS
-      (email-snapshots msg (slurp inline-file)) ; email it to the recipients
+      (spit html-file (content/snapshot-html full-snapshot)) ; create the email in a tmp file
+      (inline-css html-file inline-file) ; inline the CSS
+      (let [file-size (.length (io/file inline-file))]
+        (when (>= file-size size-limit)
+          ;; Send a Sentry notification
+          (when c/dsn ; Sentry is configured
+            (sentry/capture c/dsn {:message (str "Rendered update email is over size limit at "
+                                                 (int (Math/ceil (/ file-size 1000)))
```

Comment:
If we remove the size from the capture message all these issues will be grouped together. This can help with stats. We can move the size into the message payload.
WYT @belucid ?